provision-osx/tasks/install_homebrew.yml

```diff
@@ -0,0 +1,85 @@
+---
+
+- 
+  name: Check homebrew directory permissions
+  file:
+    path: "{{ homebrew_prefix }}"
+    owner: root
+    group: admin
+    state: directory
+    mode: 0775
+  become: yes
+
+- 
+  name: Check homebrew directory exists
+  file:
+    path: "{{ homebrew_install_path }}"
+    owner: "{{ ansible_user_id }}"
```

Comment:
Should all occurrences of `ansible_user_id` by `ansible_ssh_user` since thats what we have in our current inventory files?